src/quo2/components/avatars/icon_avatar.cljs

```diff
@@ -5,16 +5,19 @@
 
 (def sizes
   {:big 48
+   :large 32
    :medium 32
    :small 20})
 
 (defn icon-avatar
-  [{:keys [size icon color]}]
+  [{:keys [size icon color opacity]
+    :or   {opacity 20}}]
   (let [component-size (size sizes)
-        circle-color   (colors/custom-color color 50 20)
+        circle-color   (colors/custom-color color 50 opacity)
         icon-color     (colors/custom-color-by-theme color 50 60)
         icon-size      (case size
                          :big    20
+                         :large  20
```

Comment:
Adding the new option called 'large' is just to accommodate this component in the short term. It may be preferable to pass arguments for component-size and icon-size separately, will discuss.

Reply:
Thanks Eric .. If you need me to adjust it, Just let me know

Reply:
The only adjustment to `icon-avatar` made was to add opacity as a parameter. The short story regarding icon sizing is that I exported a 20x20 icon because it was identified as 20x20 in figma's layer list. Mario pointed out that it's actually 16x16, which can be seen in figma's properties. After exporting 16x16, it worked fine and didn't require any sizing changes to `icon-avatar`.